AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -71,6 +80,7 @@ final class CoreDataMeasurementStreamStorage: MeasurementStreamStorage {
     }
     
     func saveThresholdFor(sensorName: String, thresholdVeryHigh: Int32, thresholdHigh: Int32, thresholdMedium: Int32, thresholdLow: Int32, thresholdVeryLow: Int32) throws {
+        assert(isSafelyAccessed, "Func saveThresholdFor() should be acccess within accessStorage()")
```

Comment:
Don't You want to use Strings to keep those phrase and others below? if now - ok 😄

Reply:
I thought Strings are texts that we display to users, aren't they? Now I'm not sure ;D